res/css/structures/_MatrixChat.scss

```diff
@@ -76,14 +76,28 @@ limitations under the License.
     height: 100%;
 }
 
+// We'd like to remove this, but this makes matrixchat's resizehandle's
+// negative margin greater than its positive padding. If it's the same
+// or less, Safari and other WebKit based browsers get confused about overflows somehow and
+// https://github.com/vector-im/element-web/issues/19863 happens.
+.mx_MatrixChat > .mx_ResizeHandle.mx_ResizeHandle_horizontal {
+    margin: 0 calc(-5.5px - $container-gap-width / 2) 0 calc(-6.5px + $container-gap-width / 2);
+    // The condition to prevent bleeding is: (margin-left+maring-right < -11px) (IF there is NO margin on the leftPanel_wrapper)
```

Comment:
```suggestion
    // The condition to prevent bleeding is: (margin-left + margin-right < -11px) (IF there is NO margin on the leftPanel_wrapper)
```